test/ClientSpec.js

```diff
@@ -1,21 +1,21 @@
 import Client from '../lib/Client';
 import {expect} from 'chai';
 import Chance from 'Chance';
-var chance = new Chance();
+const chance = new Chance();
 
 describe('Client', () => {
 	describe('When adding a global header', () => {
-		var header;
-		var value;
+		let header;
+		let value;
 
-		beforeEach(() => {
-			header = chance.header;
-			value = chance.value;
+		before(() => {
+			header = chance.string();
+			value = chance.string();
 
-			Client.addHeader(chance.string(), chance.string());
+			Client.addHeader(header, value);
```

Comment:
This was messing up the other Helpers test before, as the headers weren't really being removed!